openedx/core/djangoapps/organization_api/api/views.py

```diff
@@ -0,0 +1,39 @@
+"""
+Organizations API views.
+"""
+from rest_framework import permissions
+from rest_framework.authentication import SessionAuthentication
+from rest_framework.response import Response
+from rest_framework.status import HTTP_404_NOT_FOUND
+from rest_framework.views import APIView
+from rest_framework_jwt.authentication import JSONWebTokenAuthentication
+from rest_framework_oauth.authentication import OAuth2Authentication
+
+from util.organizations_helpers import get_organization_by_short_name
+
+
+class OrganizationsView(APIView):
```

Comment:
I concur that edx-organizations was originally written with the idea that eventually it would become a standalone IDA, which didn't materialize, and thus there's a lot of extra stuff going on now that could likely be eliminated.  I also agree that we should keep all of the organization-related code together.